gsoc_contributors_guide.md

```diff
@@ -0,0 +1,33 @@
+# GSoC Contributor's Guide
+
+The Echoshader team aims to recruit talented [Google Summer of Code (GSoC)](https://summerofcode.withgoogle.com/) participants to help us create the capability to interactively visualize large volumes of cloud-hosted ocean sonar data to accelerate the the data exploration and discovery process. This project will go hand-in-hand with ongoing development of the [echopype](https://github.com/OSOceanAcoustics/echopype) package that handles the standardization, pre-processing, and organization of these data. We aim for creating ocean sonar data visualization functionalities based on the [HoloViz](https://github.com/holoviz/holoviz) suite of tools, and developing widgets that can be flexibly combined in [Panel](https://panel.holoviz.org/) or used in Jupyter notebooks for interactive data visuzliation.
+
+
+## Getting Started
+
+TBD
+
+
+
+
+## Brainstorm with us!
+
+In the [Issues](https://github.com/OSOceanAcoustics/echoshader/issues) section of this repository, we list some visualization ideas from mentors. We encourage you as a GSoC participant to propose your own original ideas as new issues.
+
+Feel free to propose project ideas [raising an issue](https://github.com/OSOceanAcoustics/echoshader/issues/new/choose) in this repo.
+
+After you [sign up as a GSoC participant](https://summerofcode.withgoogle.com/get-started/), feel free to to propose project ideas and ask questions via [raising an issue](https://github.com/OSOceanAcoustics/echoshader/issues/new/choose).
+
+Once the official application opens, please submit your proposals based on the [Echoshader GSoC Proposal](proposal-template.md) template.
+
+
+### Questions?
+
+For project-related question, feel free to [raise an issue] (https://github.com/OSOceanAcoustics/echoshader/issues/new/choose). 
+
+Having more questions about being a GSoC mentor or participant? Check out the [GSoC mentor & participant guides](https://google.github.io/gsocguides/).
+
+
+## The Mentor Team
+<!-- Open Source Ocean Acoustics started back in 2018 from [OceanHackWeek](https://oceanhackweek.github.io/). It is meant as a home for open source tools and resources in ocean acoustics.  -->
```

Comment:
Probably could just remove this

Reply:
i think it's good info to add, but I couldn't figure out how to absorb it coherently into this paragraph

Reply:
How about add a History section?

```
## History 

Open Source Ocean Acoustics started back in 2018 from [OceanHackWeek](https://oceanhackweek.github.io/). It is meant as a home for open source tools and resources in ocean acoustics. Currently, a core team that actively develops packages are a part of the [Echospace](https://uw-echospace.github.io/) group at the University of Washington.
```